src/utils/NucleicAcidSearchEngine.cpp

```diff
@@ -981,6 +1001,7 @@ class NucleicAcidSearchEngine :
     bool use_adducts = getFlag_("precursor:use_adducts");
     bool include_unknown_charge = getFlag_("precursor:include_unknown_charge");
     bool single_charge_spectra = getFlag_("decharge_ms2");
+    bool deisotope_spectra = getFlag_("deiosotope_ms2");
```

Comment:
```suggestion
    bool deisotope_spectra = getFlag_("deisotope_ms2");
```

Reply:
Hmm, maybe de-iOSotoping has something to do with downgrading an iphone?